dwave/cloud/auth/flows.py

```diff
@@ -202,3 +222,72 @@ def from_config_model(cls, config: ClientConfig, **kwargs) -> LeapAuthFlow:
             authorization_endpoint=authorization_endpoint,
             token_endpoint=token_endpoint,
             session_config=session_config)
+
+    def run_oob_flow(self):
+        """Run the OAuth 2.0 Authorization Code exchange flow, using the out of
+        band code exchange.
+
+        After authorizing access by visiting the authorization URL displayed,
+        user has to copy the authorization code and paste it back in their
+        terminal. On successful completion Leap API access and refresh tokens
+        are returned.
+        """
+        self.redirect_uri = self._OOB_REDIRECT_URI
+
+        url = self.get_authorization_url()
+        print(self._VISIT_AUTH_MSG, url)
+
+        code = input(self._INPUT_CODE_MSG)
+        return self.fetch_token(code=code)
+
+    def run_redirect_flow(self, *, open_browser: Union[bool,Callable] = False,
+                          timeout: Optional[float] = None):
+        """Run the OAuth 2.0 Authorization Code exchange flow, using a redirect
+        URI hosted on a local server.
```

Comment:
```suggestion
        """Run the OAuth 2.0 code exchange (using locally hosted redirect URI). 
        
        Run the OAuth 2.0 Authorization Code exchange flow, using a redirect
        URI hosted on a local server.
```

For both methods the first sentence docstring won't fit the summary rendered in HTML